client/state/notices/selectors.js

```diff
@@ -25,3 +25,8 @@ export const getNoticeLastTimeShown = createSelector(
 	( state, noticeId ) => state.notices.lastTimeShown[ noticeId ] || 0,
 	( state ) => state.notices.lastTimeShown
 );
+
+export const getNotice = createSelector(
```

Comment:
Should this be part of a separate PR?

Reply:
The change in this PR relies on `getNotice`, so no. But if we use a different approach, then we can delete it altogether!